app/controllers/passengers_controller.rb

```diff
@@ -28,9 +28,9 @@ def index
       @filters << 'active'
     end
 
-    if( params[:print].present? )
+    if params[:print].present?
       pdf = PassengersPDF.new(@passengers, @filters)
-      send_data pdf.render, filename: "Going to be the date",
+      send_data pdf.render, filename: "Current Temporary Passengers",
```

Comment:
Since people can print passengers other than the temporary passengers, why don't you just put whatever the filter is in here? Like
``` ruby
"#{@filters.join(' ') Passengers as of #{Date.today}"
```